internal/kms/kms.go

```diff
@@ -269,6 +269,26 @@ func (k *Kms) RotateKeys(ctx context.Context, scopeId string, opt ...Option) err
 	return nil
 }
 
+// ListDataKeyVersionReferencers will list the names of all tables
+// referencing the private_id column of the data key version table.
+// Supported options:
+//   - WithTx
+func (k *Kms) ListDataKeyVersionReferencers(ctx context.Context, opt ...Option) ([]string, error) {
+	const op = "kms.(Kms).ListDataKeyVersionReferencers"
+
+	opts := getOpts(opt...)
+	kmsOpts := []wrappingKms.Option{
+		// underlying key version refs function supports WithReaderWriter option, but the interfaces don't match and can't be passed
```

Comment:
Minor point: can't we convert the option like we do in RotateKeys above? I'd argue we don't need to support any options at all here until we actually have a need for them, so feel free to just remove this too.